schemas/3DScenesConfiguration/v1.0.0/3DScenesConfiguration.schema.json

```diff
@@ -236,6 +236,62 @@
                 "time"
             ]
         },
+        "IDataHistoryBasicTimeSeries": {
+            "type": "object",
+            "description": "A basic timeseries to be rendered in the chart of the data history widget",
+            "additionalProperties": false,
+            "properties": {
+                "expression": {
+                    "type": "string"
+                },
+                "unit": {
+                    "type": "string"
+                },
+                "label": {
+                    "type": "string"
+                }
+            },
+            "required": ["expression"]
+        },
+        "IDataHistoryTimeSeries": {
+            "type": "array",
+            "description": "A list of timeseries to render in the chart",
+            "items": {
+                "$ref": "#/$defs/IDataHistoryBasicTimeSeries"
+            }
+        },
+        "IDataHistoryConnectionString": {
```

Comment:
the intention behind was if we need to change the the syntax of the connection string, I wanted to keep it as a separate type. But since it is just string, we don't need it yes.

Reply:
by the way you will need to reset your config file in your container if you already created a data history widget using this scehema. otherwise it will throw error saying that parsing failed.